global-hooks/openapi/config-values.yaml

```diff
@@ -187,12 +187,12 @@ properties:
         properties:
           httpProxy:
             type: string
-            pattern: '^(http|https)://[0-9a-zA-Z\.\-:]+$'
+            pattern: '^http://[0-9a-zA-Z\.\-:]+$'
```

Comment:
This change is not valid. 

Http proxy - a proxy for HTTP requests. Https proxy - a proxy for HTTPS requests. You can still serve HTTPS requests with an HTTP proxy and vice versa.

Reply:
Thnks. Fixed.